### DISCUSSION TITLE
【質問】モータ制御に関して

### DISCUSSION THREAD

**Discussion by tsukada-shigeki** [toppers/users-forum#73] [TOPPERS Seminar]
竹内様

お世話になります。
toppers基礎実装セミナーの範囲外で申し訳ありません。

１．目的
２個のDCモータ出軸を接続します。一方に指令電圧（pwm出力）を与え回します。他方のモータからの逆起電力を検出（adc入力）し、指令電圧と逆起電力が等しくなる様にFB制御をしたい。これを例えばadc入力タスク、モータ制御タスク、ｌｃｄ表示タスクとして実現したいと思います。

２．速度制御事例
ラズパイ＋パイソンでで組んだプログラムで上記制御をしてみました。
添付ファイルにリンク先が有りますのでご覧願います。

３．質問
TOPPERS　BASE　PLATFORM　にモータ制御を組み込む方法をご教示願います。
アプローチ方法だけでも結構ですので宜しくお願い致します。
以上宜しくお願い致します。

**Comment by Ryohsuke-Takeuchi** [comment]
「モータ制御に関して」で回答。
